src/include/reconcile.i

```diff
@@ -0,0 +1,319 @@
+/*-
+ * Copyright (c) 2014-2019 MongoDB, Inc.
+ * Copyright (c) 2008-2014 WiredTiger, Inc.
+ *	All rights reserved.
+ *
+ * See the file LICENSE for redistribution information.
+ */
+
+#define	WT_CROSSING_MIN_BND(r, next_len)				\
+	((r)->cur_ptr->min_offset == 0 &&				\
+	    (next_len) > (r)->min_space_avail)
+#define	WT_CROSSING_SPLIT_BND(r, next_len) ((next_len) > (r)->space_avail)
+#define	WT_CHECK_CROSSING_BND(r, next_len)				\
+	(WT_CROSSING_MIN_BND(r, next_len) || WT_CROSSING_SPLIT_BND(r, next_len))
+
+/*
+ * __rec_need_split --
+ *	Check whether adding some bytes to the page requires a split.
+ */
+static inline bool
+__rec_need_split(WT_RECONCILE *r, size_t len)
```

Comment:
I would like to see all the functions in here prepended with `__wt_` since they're all called from many files. It at least gives an indication to the reader that it may be in a different file. If I see a call without the `__wt` prefix, I always look for it in the current file. (Like `txn.i` uses lots of `__wt` inlined functions.)

Reply:
Sure, pushed in f9fb748.